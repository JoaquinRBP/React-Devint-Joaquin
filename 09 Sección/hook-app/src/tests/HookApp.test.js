import React from 'react';
import { shallow } from 'enzyme';
import {HookApp} from '../HookApp';
describe('Pruebas sobre HookApp', () =>{
    test('Correctamente ejecutado', () =>{
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
})