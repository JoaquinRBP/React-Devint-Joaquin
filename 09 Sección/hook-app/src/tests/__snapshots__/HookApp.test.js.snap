// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre HookApp Correctamente ejecutado 1`] = `
<div>
  <p>
    Hola Mundo
  </p>
</div>
`;
